Zenventory/Views/AddProductView.swift

```diff
@@ -8,65 +8,76 @@
 import SwiftUI
 import PhotosUI
 
-
-struct AddProductView: View {
+internal struct AddProductView: View {
 
     @Environment(\.dismiss) private var dismiss
 
-    @StateObject private var vm: AddProductVModel
+    @StateObject private var addProductViewModel: AddProductViewModel
 
-    init(coreDataService: CoreDataService) {
-        _vm = StateObject(wrappedValue: AddProductVModel(dataService: coreDataService))
+    internal init(
+        coreDataService: CoreDataService
+    ) {
+        _addProductViewModel = StateObject(
+            wrappedValue: AddProductViewModel(
+                dataService: coreDataService
+            )
+        )
     }
 
     var body: some View {
         NavigationStack {
             ZStack {
-                ZColor.background
+                Color.backgroundColor()
                 VStack(spacing: 20) {
-                    PhotosPicker(selection: $vm.selectedProductPhoto, matching: .images, photoLibrary: .shared()) {
-                        if vm.productImage == nil {
-                            Image(systemName: "camera.circle.fill")
-                                .resizable()
-                                .aspectRatio(contentMode: .fit)
-                                .frame(width: 70, height: 70)
-                                .foregroundColor(ZColor.action)
-                                .background(.clear)
-                                .padding(.bottom, 50)
+                    PhotosPicker(
+                        selection: $addProductViewModel.selectedProductPhoto,
+                        matching: .images,
+                        photoLibrary: .shared()
+                    ) {
+                        if let image = addProductViewModel.productImage {
+                            Image(uiImage: image)
```

Comment:
Try to not use `Image(uiImage:)` in swiftui. But if it's necessary in this case, then it's okay :)